### PR TITLE
add variable to configure SNS sender

### PR DESCRIPTION
**Pull request by bentsku**
## Motivation
As reported by @HarshCasper, the sender of SNS email notifications was hardcoded to `admin@localstack.com` with no mean of changing that. We should replace this value by something like `no-reply@sns.localstack.cloud` or something similar, as AWS is sending those notifications with `no-reply@sns.amazonaws.com`. 

I'm a bit worried about changing this directly however, as some users might expected and test that emails are arriving from that sender. 

## Changes
- introduce `SNS_SES_SENDER_ADDRESS` config variable to make it configurable, and fallback to `admin@localstack.com` if not configured

## TODO

What's left to do:

- [x] [open a PR](https://github.com/localstack/docs/pull/1582) in the docs to document this variable and the default value

### CODE REVIEW

File: localstack-core/localstack/services/sns/publisher.py [hunk 1]
```diff
@@ -569,13 +569,16 @@ def _publish(self, context: SnsPublishContext, subscriber: SnsSubscription):
         region = extract_region_from_arn(subscriber["Endpoint"])
         ses_client = connect_to(aws_access_key_id=account_id, region_name=region).ses
         if endpoint := subscriber.get("Endpoint"):
+            # TODO: legacy value, replace by a more sane value in the future
+            #  no-reply@sns-localstack.cloud or similar
+            sender = config.SNS_SES_SENDER_ADDRESS or "admin@localstack.com"
```
Comment (alexrashed) [inline localstack-core/localstack/services/sns/publisher.py:574]: **suggestion:** ‏Would be great to have a "localstack only" test for this newly introduced config value.

**Review by alexrashed — APPROVED** [review]
LGTM! 💯 Nice and clean, would be great to have a test though, but that could also be added in a later iteration (not blocking). 😉

File: localstack-core/localstack/services/sns/publisher.py (hunk 1 above)
Comment (bentsku) [reply localstack-core/localstack/services/sns/publisher.py:574]: Very good point, I've added the test now, tell me what you think 😄

**Review by alexrashed — APPROVED** [review]
Awesome! Thanks for addressing the comment right away and adding a test! 🚀 💯 :shipit:

**Pull request merged**